src/Mod/Start/Gui/DlgStartPreferences.ui

```diff
@@ -548,6 +548,9 @@
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
+        <property name="toolTip">
+         <string>Needs a tooltip!!!</string>
```

Comment:
needs a tooltip

Reply:
This should be a tooltip for 'Use gradient for New File icon' in the start page preferences

Reply:
If this is checked, the 'New File' icon shows a gradient icon instead of the normal icon

Reply:
748755b